src/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -116,7 +116,6 @@ export class ArtworkGridContainer extends React.Component<
           <GridItem
             artwork={artwork}
             key={"artwork-" + j + "-" + artwork.__id}
-            user={this.props.user}
```

Comment:
This is removed because `GridItem` now subscribes to the context. If this is passed in, it will override what comes from the context (a surprise to me....but that is what I witnessed.)

That seems like it would be fine, because the user should always just be coming from the context...but there are several instances of `<ArtworkGrid>` that do not pass the user through. In those cases, the `user` coming from `withSystemContext` [here](https://github.com/artsy/reaction/pull/2480/files#diff-69470519ac81a7d74e2dbdb9d873891fR163) gets overwritten by `undefined`. 🤷‍♂